### PR TITLE
DW-3754 original commit

### PR DESCRIPTION
**Pull request by MattToczek**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (IanUtley) [inline public/index.html:10]: Possibly want a better description

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    />
+    <link rel="stylesheet" type="text/css" href="css/govuk-template.css">
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+
+    <!--
```
Comment (IanUtley) [inline public/index.html:18]: How is this file used, we should strip these comments.

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (IanUtley) [inline public/manifest.json:3]: Again, name and short_name need amending.

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <p>Initial Commit</p>
```
Comment (IanUtley) [inline src/App.js:8]: Why do you want to merge this into master, this application doesnt appear to do anything?

File: src/App.test.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
```
Comment (IanUtley) [inline src/App.test.js:6]: Doesnt this test fail as the App.js doesnt render a link with the text 'learn react;

File: public/index.html (hunk 1 above)
Comment (MattToczek) [reply public/index.html:10]: These are defaults that are set when setting up a React app. @HamadAli248 should have changed these in his branch - I will check. This commit was only meant to get the template on to master for Hamad to continue working on it.

File: src/App.js (hunk 4 above)
Comment (MattToczek) [reply src/App.js:8]: Again, this was just a template - Hamad has further work to commit to it

**Pull request closed without merging**